Extensions/BBText/bbtextruntimeobject.js

```diff
@@ -1,34 +1,53 @@
+//TODO @blurrymind correct the descriptions.
+/**
+ * @typedef {Object} BBTextObjectDataType Base parameters for {@link gdjs.BBTextRuntimeObject}
+ * @property {Object} content The base parameters of the BBText
+ * @property {number} content.opacity The opacity of the BBText
+ * @property {boolean} content.visible Is the text visible?
+ * @property {string} content.text Text of the object
+ * @property {string} content.color The color of the object
+ * @property {string} content.fontFamily The Font of the object
+ * @property {number} content.fontSize The size of the Text of the object
```

Comment:
extra capital

Reply:
what do you mean? which line exactly?

Reply:
"The size of the Text"

By the way, "of the object" seems redundant, let's keep it short:

` * @property {number} content.fontSize Size of the text`

Reply:
Changed